docs/dockhand.js

```diff
@@ -225,3 +246,11 @@ function headerLevel(node) {
 function debug(str) {
     console.log(str);
 }
+
+class MarkdownError extends Error {
+    constructor(file, inner) {
+        super(`failed to parse ${file}`);
```

Comment:
Do we care about the original error being eaten here?

Reply:
This is a great use case for the error cause proposal, but lacking that, you could make MarkdownError extend AggregateError instead, and preserve the original error that way?

Reply:
@ljharb @wraithgar I don't think we care that much about this until we actually run into a problem making the docs pass/build. We can backlog some work to improve this & feel free to add changes.